docs/sv_SE/faq.md

```diff
@@ -24,18 +24,18 @@ Därför är det, förutom idén om frivillighet, ingen mening för oss att ta p
 :::
 
 ### Hur kan jag bidra till LSS-Manager?
-Den "normala" användaren kan [rapportera fel][error] eller [göra förslag][suggestions].
+Den "normala" användaren kan [report bugs][error] eller [make suggestions][suggestions].
```

Comment:
@Suchty112  Why you did change this to English? This could be sv_SE. Only the name between the second `[ ]` need to be the same as in the downpart of the file.